config/routes.rb

```diff
@@ -55,6 +55,10 @@
   post '/add_comment', to: 'comments#create', as: :add_comment
   post '/remove_comment', to: 'comments#delete', as: :remove_comment
 
+  resources :citations
+  post '/add_citation', to: 'citations#add_citation', as: :add_citation
+  post '/remove_citation', to: 'citations#delete', as: :remove_citation
```

Comment:
these should be named consistently.

Reply:
i don't think i understand the feedback here. are you trying to say that throughout the app we use citation, post link, and reference interchangeably and we should stick to one of the three?

Reply:
the action name should be #remove_citation